fgcm/fgcmGray.py

```diff
@@ -1134,6 +1136,11 @@ def computeExpGrayCuts(self):
             expGrayHighCut[i] = max(0.005,
                                     min(cut, expGrayHighCut[i]*2))
 
+            self.fgcmLog.info("ExpGray cut (%s band): %.4f" % (self.fgcmPars.bands[i],
```

Comment:
Use `,` instead of `%` to avoid formatting when the log is disabled.

Reply:
There's a whole lot more that needs to be fixed in the fgcm code because of when I wrote it originally...